tests/integration/targets/utils_remove_keys/vars/main.yaml

```diff
@@ -23,3 +23,21 @@ remove:
       description: Configured by Ansible - Interface 2 (ADMIN DOWN)
       mtu: 600
       enabled: false
+  l1:
```

Comment:
@roverflow same for this as well, we can create a new file called complex.yaml which has the output of all the complex integration tests ref:  [PR](https://github.com/ansible-collections/ansible.utils/pull/200) 
CC: @NilashishC